assets/js/lesson_9/recursive_func_pow.js

```diff
@@ -0,0 +1,23 @@
+const powNumber = (num, degree) => {
+    let result
+
+    if (degree === 0) {
+        return 1
+    }
+
+    if (degree > 0) {
+        result = num * powNumber(num, degree - 1)
+    } else {
+        result = 1 / num * powNumber(num, degree + 1)
+    }
+
+    /**
+     * "powNumber(num, degree + 1) / num" also works for negative degree.
+     *  Dk if any of them more preferable to use.
+     * */ 
+    
+    return result
```

Comment:
you dont need to return `result`. It will be better to return you recursive function